packages/director/src/execution/utils.ts

```diff
@@ -9,7 +9,9 @@ export const getClaimedSpecs = (run: Run, groupId: string) =>
 export const getFirstUnclaimedSpec = (run: Run, groupId: string) =>
   getSpecsForGroup(run, groupId).find((s) => !s.claimedAt);
 export const getFailedSpecs = (run: Run, groupId: string) =>
-  getSpecsForGroup(run, groupId).filter((s) => s.results!.stats.failures > 0);
```

Comment:
@bjartur20 did you try simply replacing ! by ?
```
getSpecsForGroup(run, groupId).filter((s) => s.results?.stats.failures > 0);
```

Reply:
@bjartur20 hello?

Reply:
Yeah sorry for the late reply.
I tried that but that still resolves in a type error since s.results is possibly undefined.
I thank that my fix should cover this, since now it'll only check for the results if they are defined.

Reply:
@bjartur20 could you please provide steps to reproduce ?

Reply:
I created a bug issue which includes steps to reproduce: https://github.com/sorry-cypress/sorry-cypress/issues/652#issue-1380911426